syntax = "proto3";

package starry.examples;

// Echo 服务定义
service EchoService {
  // 简单的回显方法
  rpc Echo (EchoRequest) returns (EchoResponse);
}

// 请求消息
message EchoRequest {
  string message = 1;
}

// 响应消息
message EchoResponse {
  string message = 1;
}
